src/visual.ts

```diff
@@ -1273,15 +1273,27 @@ export class MekkoChart implements IVisual {
         let showAllDataPoints: boolean = (<BaseColumnChart>this.layers[0]).getData().showAllDataPoints;
         let defaultDataPointColor: string = (<BaseColumnChart>this.layers[0]).getData().defaultDataPointColor;
         let labelSettings: VisualDataLabelsSettings = (<BaseColumnChart>this.layers[0]).getData().labelSettings;
-
         let xAxisRotationEnabled : boolean = xAxisLabelsSettings.enableRotataion;
         let showCategoryAxis:boolean = <boolean>(this.categoryAxisProperties["show"]);
+        let showCategoryAxisTitle:boolean = <boolean>(this.categoryAxisProperties["showAxisTitle"]);
+        let categoryAxisLabelColor:string = <string>(this.categoryAxisProperties["labelColor"]);
+        let categoryAxisFontSize:number = <number>(this.categoryAxisProperties["fontSzie"]);
```

Comment:
You can use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) `??` to provide default values instead of extra if statements (this will help with the 100 line limit from eslint and make the code more readable). E.g. 
`const categoryAxisFontSize = <number>(this.categoryAxisProperties["fontSize"]) ?? 9;`

Reply:
Thank you for the feedback!